Lesson 6/src/simulation/Simulation.java

```diff
@@ -0,0 +1,178 @@
+package simulation;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import design.Item;
+import design.Rocket;
+import design.U1;
+import design.U2;
+
+public class Simulation {
+
+	public ArrayList<Item> loadItems(String file) throws FileNotFoundException
+	{
+		ArrayList<Item> items=new ArrayList<>();
+		File phase=new File(file);
+		Scanner scanner=new Scanner(phase);
+
+		while(scanner.hasNextLine())
+		{
+			String line=scanner.nextLine();
+			String[] lineSplit=line.split("=");
+
+			Item i=new Item(lineSplit[0], Integer.valueOf(lineSplit[1]));
+			items.add(i);
+
+
+		}
+		return items;
+
+	}
+
+	public ArrayList<Rocket> loadU1 (ArrayList<Item> getItems)
+
+	{  
+
+		ArrayList<Rocket> list_rocketsU1=new ArrayList<>();
+		Rocket r1=new U1();
+		for(Item h:getItems) 
```

Comment:
This for loop is wrong. Need to be fixed. 

Since we know the items inside phases-1&2 and U1 specs, with basic calculations we know that the rockets loaded should be:
- Phase-1: 9 Rockets
- Phase-2: 4 Rockets